WebServer/microservices/db/Dockerfile

```
@@ -6,7 +6,7 @@ COPY requirements.txt requirements.txt
 RUN python3 -m pip install --upgrade pip
 RUN python3 -m pip install -r requirements.txt
 
-ADD ./ /app
+ADD ./ /app/
```

```suggestion
ADD ./*.py /app/
```
Vi er vel kun interesserede i python filer. Så får vi ikke alt muligt andet med, som eks. sql filen, og Singularity definition filer